src/main/java/de/orchound/gameoflife/parsing/RleBoardParser.java

```diff
@@ -0,0 +1,60 @@
+package de.orchound.gameoflife.parsing;
+
+import de.orchound.gameoflife.game.Board;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class RleBoardParser implements BoardParser {
+
+	private Board board;
+
+	@Override
+	public Board parse(File file) {
+		try {
+			Files.lines(file.toPath()).filter(line -> !line.startsWith("#")).forEach(this::parseLine);
+		} catch (IOException e) {
+			e.printStackTrace();
```

Comment:
Just throw an `UncheckedIOException` with a useful message here. We can print it to STDERR when we start using the parser.
```suggestion
			throw new UncheckedIOException("Could not read file " + file.getPath(), e);
```